app/src/lib/stores/helpers/pull-request-updater.ts

```diff
@@ -0,0 +1,120 @@
+import { PullRequestStore } from '../pull-request-store'
+import { Account } from '../../../models/account'
+import { fatalError } from '../../fatal-error'
+import { PullRequest } from '../../../models/pull-request'
+import { GitHubRepository } from '../../../models/github-repository'
+
+const PullRequestInterval = 1000 * 60 * 3
+const StatusInterval = 1000 * 60
```

Comment:
I think we wanna bump this and `PullRequestInterval` a bit.

Maybe push them both up to 10 minutes?

Reply:
👌